tests/integration/test_charm.py

```diff
@@ -391,3 +394,26 @@ async def test_audit_log(ops_test: OpsTest) -> None:
         item = json.loads(line)
         # basic sanity check
         assert audit_log_line_sanity_check(item), "Audit sanity log check failed for first line"
+
+
+@pytest.mark.group(1)
+async def test_log_roate(ops_test: OpsTest) -> None:
+    """Test that log are being rotated."""
+    app_name = await get_app_name(ops_test)
+    await start_continous_writes(ops_test, 1)
+    # Note: this timeout out depends on max log size
+    # which is defined in "src/config.py::Config.MAX_LOG_SIZE"
+    time.sleep(60 * 10)
```

Comment:
Can we either:
1. Add a constant `TIME_TO_WRITE_100MB`

or 

2. go into the charm and **temporarily** replace the `Config.MAX_LOG_SIZE` to `1MB` for the purpose of this test, so that we do not have to wait `60*10` for the test to run? I can imagine this will clog resources

Reply:
Added constant

Reply:
Also sorry, one more question, is 1minute gaurenteed to get us to the necessary number of writes? i.e. is it possible that the test could be flakey? 

I ask because maybe we could wrap the entire test in a retry for X minutes that way it can finish earlier or later.

Reply:
Also I think it would be best if we **eventually** did something like `go into the charm and temporarily replace the Config.MAX_LOG_SIZE to 1MB for the purpose of this test, so that we do not have to wait 60*10 for the test to run? I can imagine this will clog resources`

But no need to do immediately. Would you mind creating a JIRA ticket for it and if you have time before you leave, implementing it? Otherwise I will leave it on the backlog to be addressed later on :)

Reply:
10 mins of writes using "continuous_writing" give more than 50MB of  audit.log,  another extra  minute is just for safety reason.

Adjusting size of the test will work can be done later, but it will work only for newly deployed clusters, not for existing ones.